app/views/standard/additionalMetadataStart.scala.html

```diff
@@ -1,39 +1,64 @@
-@import views.html.partials.{transferReference, progressIndicator}
-@import java.util.UUID
+@import views.html.partials.{notificationBanner, progressIndicator, transferReference, warningMessage}
 
-@(parentFolder: String, parentFolderId: UUID, consignmentRef: String, consignmentId: UUID, name: String)(implicit messages: Messages, request: RequestHeader)
+@import java.util.UUID
+@(consignmentRef: String, consignmentId: UUID, name: String, closure: List[String], descriptive: List[String])(implicit messages: Messages, request: RequestHeader)
```

Comment:
As per earlier comment maybe rename the `closure` and `descriptive` params to something a bit more descriptive.